### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by JaanviNL**
Hello Everyone ,
Please Review Alert component
alert

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap');
+*
+{
+    --error-color:#ef4444;
+    --success-color:#15803d;
+    --warning-color:#f59e0b;
+    --info-color:#06b6d4;
```
Comment (Naveenchandar) [inline components/alert/alert.css]: Suggestion:
Please avoid using color in variable names.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    margin: 1.5rem;
+}
+.alert
+{
+    display: flex;
+    justify-content: space-between;
+    width: 75%;
+   padding-left: 1rem;
+    font-weight: bold;
+    color: white;
```
Comment (Naveenchandar) [inline components/alert/alert.css:28]: Please use hex codes for consistency.

File: components/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+}
+.outline-info
+{
+color: white;
+border: none;
+background-color: var(--info-color);
+}
+.outline-warning
+{
+    border-color: var(--warning-color);;
```
Comment (Naveenchandar) [inline components/alert/alert.css]: Please remove the extra semicolon

File: components/alert/alert.css (hunk 1 above)
Comment (parthpandyappp) [reply components/alert/alert.css]: Yes, writing color would be more verbose. Avoid it.

**Review by Naveenchandar — CHANGES_REQUESTED** [review]
Please format the code.

**Comment by parthpandyappp** [comment]
Added indentations to the code, refactor the color names. Use both as a checklist for this PR, if done then it looks good to me!

File: components/alert/alert.css (hunk 1 above)
Comment (JaanviNL) [reply components/alert/alert.css]: Noted.

File: components/alert/alert.css (hunk 2 above)
Comment (JaanviNL) [reply components/alert/alert.css:28]: Okay. Thank you for pointing it out.

File: components/alert/alert.css (hunk 3 above)
Comment (JaanviNL) [reply components/alert/alert.css]: Removed. Thank You for reviewing

Comment (JaanviNL) [reply components/alert/alert.css]: Thank you For Reviewing.

**Review by Naveenchandar — APPROVED** [review]
Looks good

**Review by maitrakhatri — APPROVED** [review]
looks good to me. just one suggestion, some dimensions are in px and some in rem so it might look inconsistent on different devices. So make sure all the dimensions are only in one unit. (preferably rem)

**Pull request merged**